hw/adsp/dsp/imx8-mu.c

```diff
@@ -51,12 +51,69 @@ static void mu_write(void *opaque, hwaddr addr,
     struct adsp_io_info *info = opaque;
     struct adsp_dev *adsp = info->adsp;
     struct adsp_reg_space *space = info->space;
+    struct qemu_io_msg_irq irq;
+    uint64_t aux;
 
     log_write(adsp->log, space, addr, val, size,
```

Comment:
One more thing. Aux is not a really good name for a variable in this case.

use xsr and xcr for reading CR and SR registers.